packages/experiments/src/components/Chiclet/ChicletCard.base.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import {
+  BaseComponent,
+  css,
+  customizable,
+  classNamesFunction
+} from '../../Utilities';
+import { IChicletCardStyles, IChicletCardStyleProps, IChicletCardProps } from './ChicletCard.types';
+import { mergeStyles } from '../../Styling';
+import { Icon } from 'office-ui-fabric-react/lib/Icon';
+import { Image } from 'office-ui-fabric-react/lib/Image';
+import { TestImages } from 'office-ui-fabric-react/src/common/TestImages';
+
+const getClassNames = classNamesFunction<IChicletCardStyleProps, IChicletCardStyles>();
+
+@customizable('ChicletCardBase', ['theme'])
+export class ChicletCardBase extends BaseComponent<IChicletCardProps, any> {
+  private _classNames: { [key in keyof IChicletCardStyles]: string };
+
+  public render() {
+    const { title, ogType, description, image, imageType, imageWidth, imageHeight, imageAlt, url, onClick, className, footer, theme, getStyles } = this.props;
+    const actionable = (onClick) ? true : false;
+
+    this._classNames = getClassNames(getStyles, { theme: theme! });
+
+    // if this element is actionable it should have an aria role
+    const role = actionable ? (onClick ? 'button' : 'link') : undefined;
+    const tabIndex = actionable ? 0 : undefined;
+
+    var preview = this._renderPreviewImage(image, imageHeight, imageWidth, ogType);
+
+    return (
+      <div
+        tabIndex={ tabIndex }
+        role={ role }
+        onClick={ actionable ? this._onClick : undefined }
+        className={
+          css('ms-ChicletCard', className, mergeStyles(this._classNames.root)) }
+      >
+        <div
+          className={ mergeStyles(this._classNames.preview) }
+        >
+          { image ?
+            preview :
+            (<Image
+              src={ TestImages.documentPreviewTwo }
+              role='presentation'
+              alt=''
+            />)
+          }
+        </div>
+        <div
+          className={ mergeStyles(this._classNames.info) }
+        >
+          <div
+            className={ mergeStyles(this._classNames.title) }
+          >
+            { title ? title : "Placeholder" }
+          </div>
+          <div
+            className={ mergeStyles(this._classNames.link) }
+          >
+            { url ? url : "https://onedrive.com/files/v-lygi/39192908430" }
```

Comment:
>https://onedrive.com/files/v-lygi/39192908430 [](start = 27, length = 45)

Remove this URL before merging the PR

Reply:
Nope! Good catch. It was a placeholder for my testing purposes, so I'll remove it.